src/Node.hs

```diff
@@ -98,7 +100,7 @@ listenerMessageName (ListenerActionConversation f) =
                 -> Proxy rcv
                 -> MessageName
         msgName _ = messageName
-    in  msgName (f undefined) Proxy
+    in  msgName (f undefined undefined) Proxy
```

Comment:
Probably better replaced with Proxy constructors. But given we never actually pattern-match on proxy, these are kind of equal solutions